SwrveSDKTest/src/test/java/com/swrve/sdk/utils/TestHttpServer.java

```diff
@@ -0,0 +1,99 @@
+package com.swrve.sdk.utils;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+
+import fi.iki.elonen.NanoHTTPD;
+
+public class TestHttpServer extends NanoHTTPD {
```

Comment:
I don't like using this in unit tests. Think its better to mock the restclient like here:
https://github.com/Swrve/swrve-android-sdk/blob/release-4_6/SwrveSDKTest/src/test/java/com/swrve/sdk/SwrveInAppMessageActivityTest.java#L54